lib/pengine/unpack.c

```diff
@@ -3758,6 +3771,18 @@ unpack_rsc_op(pe_resource_t *rsc, pe_node_t *node, xmlNode *xml_op,
         }
     }
 
+    if (ignore_mask_probe_failure) {
+        crm_notice("Ignoring masked probe failure from resource %s on %s",
+                   rsc->id, node->details->uname);
+        update_resource_state(rsc, node, xml_op, task, target_rc, *last_failure,
+                              on_fail, data_set);
+        crm_xml_add(xml_op, XML_ATTR_UNAME, node->details->uname);
+
+        record_failed_op(xml_op, node, rsc, data_set);
+        resource_location(parent, node, -INFINITY, "hard-error", data_set);
```

Comment:
"hard-error" here is just a tag to use in trace log messages, so something unique and descriptive is warranted (e.g. "masked-probe-failure")

Reply:
Fixed.